### ISSUE TITLE
IPA and Ivona Voices

### ISSUE THREAD

**Issue by Tkael** [EDCD/EDDI#297]
## EDDI version in which issue found
[2.4.6-b1]

## Steps to reproduce

If you set a phonetic / spoken name for your ship, it is not spoken correctly via the 'Hear it' button or scripts like `{ShipName()} stands ready`. **However,** this behavior appears to be voice dependent. With Microsoft Hazel, the name set by Name (spoken) ***is*** spoken correctly. With Ivona Amy, the correct name ***is not*** used.

## Expected
I should hear the phonetic name spoken.

## Observed
I hear the standard name spoken.

## Investigation
SSML output used for testing:
```
<?xml version="1.0" encoding="UTF-8"?><speak version="1.0" xmlns="http://www.w3.org/2001/10/synthesis" xml:lang="en-GB"><phoneme alphabet="ipa" ph="praɪdɒvˈakɜːnɑ">Little Boy</phoneme> stands ready.</speak> 
```
Result with Microsoft Hazel `Pride of Achenar stands ready` (IPA is used)
Result with Ivona Amy: `Little Boy stands ready` (IPA is not used)

This issue seems to come down to the `speak version="1.0"` tag in the above SSML. 
Ivona is [compliant to SSML version 1.1](http://developer.ivona.com/en/ttsresources/ssml/ssml.html#), and defaults to 1.1 when this attribute is undefined. 

It does not, however, seem to fully support IPA when version 1.0 is specified. With another application that was bundled with my copy of Ivona Amy (MiniSpeech): 
- IPA is parsed correctly when the version is set to "1.1", an empty string, or removed entirely. 
- IPA is not parsed correctly when the version is set to "1.0"

The Microsoft Speech Synthesis engine used by EDDI [supports version 1.0](https://msdn.microsoft.com/en-us/library/vs/alm/hh361611(v=office.14).aspx) and does not support version 1.1. Consequently, EDDI throws exceptions and fails to parse IPA correctly whenever the version is set to "1.1", an empty string, or removed entirely. 

It may not be possible to resolve this fundamental incompatibility without abandoning the Microsoft Speech Synthesis engine (and that's not something we can do lightly).

**Comment by Tkael** [comment]
I've left a comment explaining this issue with the company where I purchased my copy of Ivona Amy. Awaiting their response.

**Comment by Tkael** [comment]
Response is that Ivona publishes updates about once per year and that we should wait to see if it's been fixed in the next release.

**Comment by Tkael** [comment]
Closed as unactionable.

**Comment by richardbuckle** [comment]
Reopening. TK, please can you DM me contact details as I too would like to add myt voice as a customer.

**Comment by Tkael** [comment]
Adding as an example a self-contained method which exhibits the issue:
```cs
        private static void SystemSpeechTest()
        {
            var desiredVoice = "Amy";

            // Set up our speech synthesizer and voice.
            var synth = new System.Speech.Synthesis.SpeechSynthesizer();
            var availableVoices = synth.GetInstalledVoices().Select(v => v.VoiceInfo).ToList();
            Console.Write("Available voices:\n");
            foreach (var voice in availableVoices)
            {
                Console.Write($"{voice.Culture.IetfLanguageTag} {voice.Name}\n");
            }
            synth.SelectVoice(availableVoices.Select(v => v.Name).First(v => v.Contains(desiredVoice)));
            var lang = synth.Voice.Name.Contains("CereVoice") ? synth.Voice.Culture.TwoLetterISOLanguageName : synth.Voice.Culture.IetfLanguageTag; 
            
            // Set up the string to speak and its SSML customizations.
            var xml = @"<?xml version='1.0' encoding='UTF-8'?>";
            var speakHeaders = $@"<speak version='1.0' xmlns='https://www.w3.org/2001/10/synthesis' xmlns:xsi='http://www.w3.org/2001/XMLSchema-instance' xsi:schemaLocation='http://www.w3.org/TR/speech-synthesis/synthesis.xsd' xml:lang='{lang}'>";
            var speakBody = @"You say <phoneme alphabet='ipa' ph='pɪˈkɑːn'>pecan</phoneme>. I say <phoneme alphabet='ipa' ph='ˈpi.kæn'>pecan</phoneme>.";
            var speakFooters = @"</speak>";

            var speech = xml + speakHeaders + speakBody + speakFooters;
            Console.Write($"Speaking:\n{speech}\n");

            // Generate the audio stream from the string.
            synth.SetOutputToDefaultAudioDevice();

            if (synth.Voice.Name.Contains("CereVoice"))
            {
                synth.Speak(speech);
            }
            else
            {
                synth.SpeakSsml(speech);
            }
        }
```
It should pronounce the word "pecan" in two different ways (and works successfully with Microsoft Hazel Desktop, Microsoft Zira Desktop, CereVoice Heather, and CereVoice Caitlin). It fails with Ivona Amy, instead repeating the same pronunciation regardless of the IPA spelling.

For reference, the culture of each tested voice is as follows:
en-GB IVONA 2 Amy OEM
en-GB CereVoice Caitlin - English (Ireland)
en-GB CereVoice Heather - English (Scotland)
en-GB Microsoft Hazel Desktop
en-US Microsoft Zira Desktop

**Comment by Tkael** [comment]
Resolved by #2222, which documents methods for users to correct Ivona pronunciation using the Lexicon Editor utility.
